blocks/leadspace/leadspace.css

```diff
@@ -16,22 +16,69 @@ main div.leadspace-wrapper {
     padding: 0 var(--spacer-layout-02) 0;
 }
 
-.leadspace > div {
+.leadspace:not(.document) > div {
     width: 100%;
     display: flex;
     flex-flow: column;
     gap: var(--spacer-layout-01);
 }
 
+.leadspace.document {
+    padding-bottom: var(--spacer-layout-05);
+    padding-top: var(--spacer-layout-05);
+}
+
 /* Column Specific Styles */
+/* stylelint-disable-next-line no-descending-specificity */
 .leadspace > div > div {
     position: relative;
 }
 
-.leadspace > div > div:nth-child(1) {
+.leadspace:not(.document) > div > div:nth-child(1) {
     margin: auto;
 }
 
+.leadspace.document .document-tag-container {
```

Comment:
@amol-anand This document variant you've added isn't included as part of the Leadpapce design in Figma. Have you confirmed this variant should be included as part of the LEadspace block vs a different block that is unique to the document viewer page?

Reply:
good point. it looks and functions exactly like the Leadspace which is why it makes sense as compared to a separate Leadspace just for document pages. I'll check with the Design team anyways.